File: DomandaOnLine/Logger/Data/Entity.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Logger.Data
{
	public class Entity
	{
		public Entity(){

		}

		public Entity(int id, string name){
			Id = id;
			Name = name;
		}
		public int Id { get; set; }
		public string Name { get; set; }
	}
}

File: HeliosEsercizio/Logger/Data/Entity.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Namespace Data
    Public Class Entity
        Public Sub New()
        End Sub

        Public Sub New(ByVal id As Integer, ByVal name As String)
            id = id
            name = name
        End Sub

        Public Property Id As Integer
        Public Property Name As String
    End Class
End Namespace
